app/javascript/dashboard/components/CustomAttribute.vue

```diff
@@ -6,6 +6,7 @@
           <input
             v-model="editedValue"
             class="checkbox"
+            :class="{ 'checkbox--rtl': isRTLView }"
```

Comment:
Do we need this? We can write the rule to apply any style if there's `app-rtl--wrapper` on its parents?

.app-rtl--wrapper .checkbox {
   margin-left: var(--space-small);
}